tools/genomespace/genomespace_importer.py

```diff
@@ -192,7 +191,7 @@ def download_from_genomespace_importer( username, token, json_parameter_file, ge
         # if using tmp file, move the file to the new file path dir to get scooped up later
         if using_temp_file:
             original_filename = filename
-            filename = ''.join( c in VALID_CHARS and c or '-' for c in filename )
+            filename = ''.join( c in FILENAME_VALID_CHARS and c or '-' for c in filename )
```

Comment:
Hrmm.  The valid chars sets the genomespace tool uses is indeed different.  Should this be the same set of chars, or not?  And, if so, which chars?